src/app/invitations/invitation-form/invitation-form.component.scss

```diff
@@ -1,4 +0,0 @@
-:host ::ng-deep .p-checkbox, :host ::ng-deep .p-checkbox-box {
```

Comment:
You should not use ::ng-deep.

Reply:
I'm aware of this, also mentioned in https://angular.io/guide/component-styles#deprecated-deep--and-ng-deep.
Unfortunately they have deprecated it without providing an alternative AFAIK.

In any case, I've removed and moved it to global styles.scss :)